### PR TITLE
Master

### PR DESCRIPTION
**Pull request by leitinha**

### CODE REVIEW

File: lproj/gl.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+"Locate" = "Ubicar";
+"Error retrieving RSS feed:"="Error recuperando suscripción";
+"Got HTTP status 304 - No news from last check" = "Recibido estado HTTP 304 - No hay noticias desde la última búsqueda";
+"Error retrieving RSS Icon:" = "Error recuperando icono de la suscripción:";
+"RSS Icon not found!" = "¡Icono de la suscripción no encontrado!";
+
+/*Added in 3.0.5 */
+"Redirection attempt treated as temporary for safety concern" = "Redirection attempt treated as temporary for safety concern";
+
+    Status API Training Shop Blog About Help
```
Comment (barijaona) [inline lproj/gl.lproj/Localizable.strings]: Probably a copy/paste error ?

**Comment by leitinha** [comment]
Sorry! I I'm having trouble dealing with the GitHub Desktop app. I've already edited the file (online).

**Comment by barijaona** [comment]
@leitinha : merge ViennaRSS’s master into your own master.

From now on, avoid working directly on you master. Create a new branch when you begin something new.

**Comment by leitinha** [comment]
Ok! Sorry to bother you

**Comment by barijaona** [comment]
You didn't bother me at all :smile:

It was just an advice to avoid you facing problems with merges.

**Comment by josh64x2** [comment]
Linking this to issue #485

**Pull request merged**